### PR TITLE
fixes #10 the menu is now cadetblue when in dev mode

### LINKED ISSUE #10: Staging deployment should have a different look and feel (to avoid confusion)
The left menu bar should have a different color when the environment is staging.

### PR DESCRIPTION
**Pull request by louhibi**

### CODE REVIEW

**Comment by louhibi** [comment]
image

File: souschef/sous_chef/templates/base.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <div class="pusher">
         <div class="ui grid container">
             <!-- Non responsive menu -->
-            <div class="ui left fixed vertical inverted menu">
+            <div class="ui left fixed vertical inverted menu {% if SC_ENVIRONMENT_NAME == 'DEV' %}dev-color{% endif %}">
```
Comment (guillaumep) [inline souschef/sous_chef/templates/base.html]: It's for staging that we want a different color.

**Review by guillaumep — CHANGES_REQUESTED** [review]
Some adjustments still required.

**Review by guillaumep — APPROVED** [review]
👍

**Pull request merged**